CRM/Sepa/BAO/SEPAMandate.php

```diff
@@ -895,6 +895,39 @@ public static function getRecurringContributionMandateID($recurring_contribution
     return $mandate_by_rcontribution_id[$recurring_contribution_id];
   }
 
+  public static function getLastMandateOfContact($cid) {
+    if (empty($cid)) {
```

Comment:
Please cast to ``int`` to avoid SQL injections. I don't know where it's coming from, but just to be safe...